Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -84,10 +84,10 @@
           Slash: 1
 
 - type: entity
-  name: ablative shield
+  name: laser shield
   parent: [ BaseShield, BaseRestrictedContraband ]
   id: RiotLaserShield
-  description: An ablative shield built for withstanding lasers, but not much else.
+  description: A shield built for withstanding lasers.
```

Comment:
Again like the other pr, leave in the not much else. Just add resistant to it too "laser resistant shield"